examples/biobert_ner/bio_ner_predictor.py

```diff
@@ -105,58 +105,83 @@ def predict(self, data_batch: Dict[str, Dict[str, List[str]]]) \
 
         return pred
 
-    def _fill_sub_entities(self, entity_groups_disagg, data_pack, tids):
-        first_idx = entity_groups_disagg[0]['idx']
-        first_tid = entity_groups_disagg[0]['tid']
-        while first_idx > 0 and data_pack.get_entry(first_tid).is_subword:
-            first_idx -= 1
-            first_tid -= 1
-
-        last_idx = entity_groups_disagg[-1]['idx']
-        last_tid = entity_groups_disagg[-1]['tid']
-
-        # TODO: this piece of code guesses the internal id assignment, this
-        #  could be unstable. Temporary work around.
-        try:
-            while last_idx < len(tids) and data_pack.get_entry(last_tid + 1) \
-                    .is_subword:
-                last_idx += 1
-                last_tid += 1
-        except KeyError:
-            print('Encounter get entry error.')
+    def _complete_entity(self,
+                         subword_entities: List[Dict[str, Any]],
+                         data_pack: DataPack,
+                         tids: List[int]) -> Tuple[int, int]:
+        """ Complete entity span from predicted subword entities
 
-        return first_tid, last_tid
+        Start from the first subword with predicted entity. If this entity
+        is a subword (e.g. "##on"), then move on to the previous subword until
```

Comment:
It looks like the semantic of `is_subword` is actually `is_not_first_segment`. Maybe we can rename this attribute to make it more intuitive? Or we can use `is_first_segment` with reversed boolean values?

Reply:
Makes sense. I can change the name